src/rp/index.ts

```diff
@@ -121,6 +127,85 @@ export class RelyingPartySolution {
     );
   }
 
+  /**
+   * Prepare the Verified Presentation token for a received request object in the context of an authorization request flow.
+   * The presentation is prepared by disclosing data from provided credentials, according to requested claims
+   * Each Verified Credential come along with the claims the user accepts to disclose from it.
+   *
+   * The returned token is unsigned (sign should be apply by the caller).
+   *
+   * @todo accept more than a Verified Credential
+   *
+   * @param requestObj The incoming request object, which the requirements for the requested authorization
+   * @param presentation The Verified Credential containing user data along with the list of claims to be disclosed.
+   * @returns The unsigned Verified Presentation token
+   * @throws {ClaimsNotFoundBetweenDislosures} If the Verified Credential does not contain one or more requested claims.
+   *
+   */
+  prepareVpToken(
+    requestObj: RequestObject,
+    [vc, claims]: Presentation // TODO: support multiple presentations
+  ): string {
+    // this throws if vc cannot satisfy all the requested claims
+    const vp = disclose(vc, claims);
+
+    // TODO: check all requeste claims of the requestedObj are satisfied
```

Comment:
can we put the corresponding jira task id next to each todo?

Reply:
done in https://github.com/pagopa/io-react-native-wallet/pull/20/commits/8cb6547